app/src/main/res/layout/activity_chat.xml

```diff
@@ -42,15 +42,16 @@
 
         <EditText
             android:id="@+id/newMessageText"
-            android:layout_width="339dp"
+            android:layout_width="fill_parent"
```

Comment:
Cirrus says that is it better to use layout_width of 0dp for performance reasons.
```suggestion
            android:layout_width="0dp"
```